Find a functionally equivalent code:android.widget.TextView.setError:COMMENT
Method Modifier: public      
Comment:/**
 * Sets the right-hand compound drawable of the TextView to the "error"
 * icon and sets an error message that will be displayed in a popup when
 * the TextView has focus.  The icon and error message will be reset to
 * null when any key events cause changes to the TextView's text.  If the
 * <code>error</code> is <code>null</code>, the error message and icon
 * will be cleared.
 */

@@ -2,8 +2,8 @@
     if (error == null) {
         setError(null, null);
     } else {
-        Drawable dr = getContext().getResources().getDrawable(com.android.internal.R.drawable.indicator_input_error);
+        Drawable dr = getContext().getDrawable(com.android.internal.R.drawable.indicator_input_error);
         dr.setBounds(0, 0, dr.getIntrinsicWidth(), dr.getIntrinsicHeight());
         setError(error, dr);
     }

